feat(serverless): faq small fixes (#3406)
scaleway/docs-content: faq/serverless-functions.mdx

## Can I allow to list the IPs of my functions?

> Serverless Functions does not yet support Private Networks. However, you can use the Scaleway IP ranges defined  at [https://www.scaleway.com/en/peering/](https://www.scaleway.com/en/peering/)  on Managed Databases and other products that allow IP filtering.

Reviewer: ```suggestion
Serverless Functions does not yet support Private Networks. However, you can use the Scaleway IP ranges defined  at [https://www.scaleway.com/en/peering/](https://www.scaleway.com/en/peering/) on Managed Databases and other products that allow IP filtering.
```

> Serverless Functions does not support Virtual Private Networks yet but you can use the Scaleway IP ranges defined in the following link that can be used on Managed Databases and other products that allow IP Filtering: https://www.scaleway.com/en/peering/](https://www.scaleway.com/en/peering/).

Reviewer 2: ```suggestion
Serverless Functions does not yet support Private Networks. However, you can use the Scaleway IP ranges defined  at [https://www.scaleway.com/en/peering/](https://www.scaleway.com/en/peering/)  on Managed Databases and other products that allow IP filtering.
```
